src/witan/models/dem/ccm/core/projection_loop.clj

```diff
@@ -144,5 +144,7 @@
   "Returns the population field"
   {:witan/name :ccm-core-out/population
    :witan/version "1.0.0"
-   :witan/input-schema {:population PopulationSchema}}
+   :witan/input-schema {:population PopulationSchema :net-migration NetMigrationSchema
+                        :births-by-age-sex-mother BirthsAgeSexMotherSchema
+                        :births BirthsBySexSchema :deaths DeathsOutputSchema}}
```

Comment:
these look good to me @seb231 - good call on included both forms of birth data. We could add more too e.g. full migration projections or projected rates, but I think this is great for starters.